runners/core-construction-java/src/test/java/org/apache/beam/runners/core/construction/graph/GreedyPipelineFuserTest.java

```diff
@@ -178,6 +179,7 @@ public void singleEnvironmentBecomesASingleStage() {
    * becomes all runner-executed
    */
   @Test
+  @Ignore
```

Comment:
This test needs to be fixed instead of ignored.

Reply:
agreed, sorry I tried to explain in the PR description that I needed some guidance on how to fix them. can discuss here or in slack if it's easier…

Reply:
Taking a deeper look to see what kind of guidance I can provide to fix up the tests.